pkg/client/core/api/direct/direct_clients.go

```diff
@@ -4,19 +4,6 @@ import "github.com/oslokommune/okctl/pkg/client"
 
 // ToolChain contains the direct clients
 type ToolChain struct {
-	AppPostgresIntegration client.ApplicationPostgresAPI
-	Certificate            client.CertificateAPI
-	Cluster                client.ClusterAPI
-	Component              client.ComponentAPI
-	ContainerRepo          client.ContainerRepositoryAPI
-	ExternalDNS            client.ExternalDNSAPI
-	Helm                   client.HelmAPI
-	IdentityManager        client.IdentityManagerAPI
-	ManagedPolicy          client.ManagedPolicyAPI
-	Manifest               client.ManifestAPI
-	Parameter              client.ParameterAPI
-	SecuityGroup           client.SecurityGroupAPI
-	ServiceAccount         client.ServiceAccountAPI
-	Vpc                    client.VPCAPI
-	Domain                 client.DomainAPI
+	Helm   client.HelmAPI
+	Domain client.DomainAPI
```

Comment:
Why is domain still in here?

Reply:
It does not follow the same pattern as the other direct clients. It will have a separate PR